### DISCUSSION TITLE
horizontal scrollbar

### DISCUSSION THREAD

**Discussion by frankvp11** [zauberzeug/nicegui#1150] [Ideas / Feature Requests]
I was wondering if we could get a horizontal scrollbar? Essentially, it would be able to be put in any place necessary and allows the user to scroll through a list of things horizontally. It would be much like the regular horizontal scrollbar that modern browsers have, however it would be able to be placed anywhere. 
Does something like this exist, and if so, are there any examples on it? I've already tried the changing the infinite scroll example with no luck. 
Thanks in advance

**Comment by falkoschindler** [comment]
Hi @frankvp11, do you mean something like [Quasar's Scroll Area](https://quasar.dev/vue-components/scroll-area/) we integrated in PR https://github.com/zauberzeug/nicegui/pull/1072? It will be part of our very next release. 🙂

**Comment by atticus-lv** [comment]
hi @falkoschindler ,is this feature release? I am not familiar with Quasar, can you give a example for a simple horizontal scrollbar?.

**Reply by falkoschindler** [reply]
See https://nicegui.io/documentation/scroll_area.

Here is an example for a horizontal scroll bar:
```py
with ui.scroll_area().classes('bg-blue-100 w-40 h-20'):
    with ui.row(wrap=False):
        ui.button('A')
        ui.button('B')
        ui.button('C')
        ui.button('D')
```
